### ISSUE TITLE
Failed to training with xgboost conda

### ISSUE THREAD

**Issue by Shindora** [dmlc/xgboost#8030]
/home/vi/baseline/lightgbm_xgboost.ipynb Cell 23' in <cell line: 2>()
      [1](vscode-notebook-cell://ssh-remote%2B44.232.224.181/home/vi/baseline/lightgbm_xgboost.ipynb#ch0000022vscode-remote?line=0) xg_clf = xgb.XGBClassifier()
----> [2](vscode-notebook-cell://ssh-remote%2B44.232.224.181/home/vi/baseline/lightgbm_xgboost.ipynb#ch0000022vscode-remote?line=1) xg_clf.fit(X_train,y_train)

File ~/miniconda3/envs/ds/lib/python3.8/site-packages/xgboost/core.py:532, in _deprecate_positional_args.<locals>.inner_f(*args, **kwargs)
    530 for k, arg in zip(sig.parameters, args):
    531     kwargs[k] = arg
--> 532 return f(**kwargs)

File ~/miniconda3/envs/ds/lib/python3.8/site-packages/xgboost/sklearn.py:1382, in XGBClassifier.fit(self, X, y, sample_weight, base_margin, eval_set, eval_metric, early_stopping_rounds, verbose, xgb_model, sample_weight_eval_set, base_margin_eval_set, feature_weights, callbacks)
   1377     params["num_class"] = self.n_classes_
   1379 model, metric, params, early_stopping_rounds, callbacks = self._configure_fit(
   1380     xgb_model, eval_metric, params, early_stopping_rounds, callbacks
   1381 )
-> 1382 train_dmatrix, evals = _wrap_evaluation_matrices(
   1383     missing=self.missing,
   1384     X=X,
   1385     y=y,
   1386     group=None,
   1387     qid=None,
   1388     sample_weight=sample_weight,
   1389     base_margin=base_margin,
   1390     feature_weights=feature_weights,
...
  [bt] (6) /home/vi/miniconda3/envs/ds/lib/python3.8/lib-dynload/_ctypes.cpython-38-x86_64-linux-gnu.so(+0x13c95) [0x7faa9551ac95]
  [bt] (7) /home/vi/miniconda3/envs/ds/bin/python(_PyObject_MakeTpCall+0x3bf) [0x55f9eab4813f]
  [bt] (8) /home/vi/miniconda3/envs/ds/bin/python(_PyEval_EvalFrameDefault+0x5434) [0x55f9eabf2dd4]

**Comment by trivialfis** [comment]
Could you please share the complete log and a reproducible example?

**Comment by trivialfis** [comment]
Closing due to lack of response.
